joplin/pages/topic_collection_page/factories.py

```diff
@@ -59,38 +75,40 @@ def create_topic_collection_page_from_page_dictionary(page_dictionary, revision_
     if page:
         return page
 
-
     # since we don't have a page matching the revision id, we should look
-    # for other matches, for now let's just use slug
+    # for other matches, for now let's just use the english slug
     # todo: figure out what we want the logic around importing a page with the same slug to look like
     try:
-        page = TopicCollectionPage.objects.get(slug=page_dictionary['slug'])
+        page = TopicCollectionPage.objects.get(slug=page_dictionaries['en']['slug'])
```

Comment:
non-english slug would be unique too, right?

Reply:
we don't require a non-english slug, so it's possible to get a page without one (which our API will probably return as the english one at the moment)

Reply:
By non-english, I meant `page_dictionary['slug']`. I was just wondering because I was writing some stuff that assumed `page_dictionary['slug']` would always exist and be unique. I think that's still true.